optuna/distributions.py

```diff
@@ -143,6 +143,14 @@ def _contains(self, param_value_in_internal_repr):
         else:
             return self.low <= value and value < self.high
 
+    def _asdict(self):
```

Comment:
Sorry for the lack of explanation. `BaseDistribution` already have [the default `_asdict` implementation](https://github.com/optuna/optuna/blob/4ad45e8086b09f9e8733dca90f463d433ef10288/optuna/distributions.py#L83), so you don't need to implement it for each distribution class.

Reply:
oops. Fixed it.